### DISCUSSION TITLE
Just updated neogit and telescope doesn't work

### DISCUSSION THREAD

**Discussion by marcdsdavid** [NeogitOrg/neogit#666] [Q&A] [answered]
Hello, hopefully this is the right place to post this. I updated neogit today via Lazy and it seems that telescope no longer works. Not sure if it's something with my config as it was working previously (i.e. when selecting branches).

here is my config, heavily inspired by @CKolkey
```
neogit.setup {
  kind                        = "tab",
  console_timeout             = 10000,
  auto_show_console           = true,
  disable_hint                = true,
  disable_commit_confirmation = true,
  ignored_settings            = {
    "NeogitPushPopup--force-with-lease",
    "NeogitPushPopup--force",
    "NeogitCommitPopup--allow-empty",
  },
  commit_popup                = {
    kind = "auto"
  },
  signs                       = {
    section = { "", "" },
    item    = { "", "" },
  },
}
```

and this is my Lazy:

```	
{
		'NeogitOrg/neogit',     -- git interface
		dependencies = {
			'nvim-lua/plenary.nvim',
			'sindrets/diffview.nvim',
			'nvim-telescope/telescope.nvim'
		},
	},
```
	
from my understanding of the new updates integrations is no longer needed as they are auto-detected. my diffview works without problem.

**Comment by ten3roberts** [comment, marked as answer]
https://github.com/NeogitOrg/neogit/blob/805a7aa3e5412bbc05c88fb97bef2bc219441098/lua/neogit/config.lua#L156

Could you do `export NEOGIT_LOG_CONSOLE="sync"` and then open neovim?

I am getting telescope to work with both `integrations =  { telescope  = nil }` and no integrations in my config

**Reply by marcdsdavid** [reply]
This is my log

```
[neogit] [INFO  13:52:42] /Users/marcdavid/.local/share/nvim/lazy/neogit/lua/neogit/status.lua:404: [STATUS BUFFER]: Starting refresh
[neogit] [INFO  13:52:42] /Users/marcdavid/.local/share/nvim/lazy/neogit/lua/neogit/status.lua:440: [STATUS BUFFER]: Finished refresh
[neogit] [INFO  13:52:42] /Users/marcdavid/.local/share/nvim/lazy/neogit/lua/neogit/status.lua:444: [STATUS BUFFER]: Refresh lock is now free
[neogit] [INFO  13:52:51] /Users/marcdavid/.local/share/nvim/lazy/neogit/lua/neogit/config.lua:162: [CONFIG] Found auto integration '"telescope" = true'
[neogit] [INFO  13:53:00] /Users/marcdavid/.local/share/nvim/lazy/neogit/lua/neogit/status.lua:404: [STATUS BUFFER]: Starting refresh
[neogit] [INFO  13:53:00] /Users/marcdavid/.local/share/nvim/lazy/neogit/lua/neogit/status.lua:440: [STATUS BUFFER]: Finished refresh
[neogit] [INFO  13:53:00] /Users/marcdavid/.local/share/nvim/lazy/neogit/lua/neogit/status.lua:444: [STATUS BUFFER]: Refresh lock is now free
```

**Reply by marcdsdavid** [reply]
I am sorry it's actually just not using the telescope ui-select theme as before but it is actually using telescope my bad.

**Reply by CKolkey** [reply]
Ah, yeah, it's a custom integration - you can pass a limited config to it in our setup, or disable it to use your vim.ui.select function (but some things wont work if you do)

**Reply by marcdsdavid** [reply]
got it thanks for the help!

**Reply by CKolkey** [reply]
No worries :)

**Comment by ten3roberts** [comment]
Additonally, may I ask why you decided to open this as a discussion rather than an issue?

Is it because of the issue templates that creates too much initial friction overcome when posting?

**Reply by marcdsdavid** [reply]
Sorry, I didn't create an issue straight away as I wasn't sure if it was just me with a bad configuration.

**Comment by CKolkey** [comment]
Probably has to do with lazy loading? If `pcall(require, "telescope.nvim")` fails then the integration isnt auto-detected.

**Reply by ten3roberts** [reply]
I think `Lazy.nvim` will shim `require` and replace is so that the plugin is auto-loaded upon the first require?

**Reply by CKolkey** [reply]
Yeah, that could be. Just postulating what could be up
